test/integration/service/loadbalancer_test.go

```diff
@@ -333,41 +307,55 @@ func Test_ServiceLoadBalancerEnableLoadBalancerClassThenUpdateLoadBalancerClass(
 		},
 	}
 
-	service, err = client.CoreV1().Services(ns.Name).Create(ctx, service, metav1.CreateOptions{})
+	service, err = client.CoreV1().Services(ns.Name).Create(context.TODO(), service, metav1.CreateOptions{})
 	if err != nil {
 		t.Fatalf("Error creating test service: %v", err)
 	}
 
-	if len(cloud.Calls) > 0 {
```

Comment:
same comment here, don't we want to check this behavior?

Reply:
Same above